nsxt/utils.go

```diff
@@ -662,3 +662,22 @@ func handlePagination(lister func(*paginationInfo) error) (int64, error) {
 
 	return total, nil
 }
+
+func getContextSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Description: "Resource context",
+		Optional:    true,
+		MaxItems:    1,
+		ForceNew:    true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"project_id": {
```

Comment:
Not sure whether we should use `project_path` for consistency. This would eliminate the need to add `org` in future, but might be cumbersome if projects are hard-coded rather than specified by dependency.